google/earthenterprise
# Fix #643: Version Errors With Centos6

Python 2.6 doesn't have the same API for Python's subprocess functionality; switch to using the more appropriate and flexible GitPython module instead.

WIP: This change requires GitPython to be added to the installed dependencies.

WIP: If anyone has any ideas for how to immediately get to the root of the git repository tree (required by git.Repo()), I would like a cleaner method for that than "search all parents of the current directory." But, this should just be a minor cleanliness note.

Test Procedure:
- Build with Centos 6 and 7
- Examine src/NATIVE*/gee_long_version.txt, and check that it has an "alpha" entry, or similar.

Centos 6: Checking that the fix works with Python 2.6.
Centos 7: Checking that the fix works with Python 2.7 (which uses a different version of GitPython, which in turn changes the default setting of an important parameter).
